### ISSUE TITLE
Error during compilation  

### ISSUE THREAD

**Issue by Decntdefalt** [DivanteLtd/magento2-vsbridge-indexer#5]
How to purge this module completely?

I have tried removing the folder, removing anything named magento2-vsbridge-indexer from composer.json and composer.lock

I have done module:disable.

Was a bit fast without reading that it wasn't tested with 2.3... Now there's a lot of issues.

Edit: 

```
Errors during compilation:
	Divante\VsbridgeIndexerCatalog\Model\ResourceModel\Product\AttributeDataProvider
		Incompatible argument type: Required type: \Divante\VsbridgeIndexerCatalog\Model\ResourceModel\entityType. Actual type: string; File: 
/opt/bitnami/apps/magento/htdocs/vendor/divante/magento2-vsbridge-indexer/src/module-vsbridge-indexer-catalog/Model/ResourceModel/Product/AttributeDataProvider.php

	Divante\VsbridgeIndexerCatalog\Model\ResourceModel\Category\AttributeDataProvider
		Incompatible argument type: Required type: \Divante\VsbridgeIndexerCatalog\Model\ResourceModel\entityType. Actual type: string; File: 
/opt/bitnami/apps/magento/htdocs/vendor/divante/magento2-vsbridge-indexer/src/module-vsbridge-indexer-catalog/Model/ResourceModel/Category/AttributeDataProvider.php

Total Errors Count: 2

In Log.php line 92:
                            
  Error during compilation
```

**Comment by dennisotugo** [comment]
+1

**Comment by afirlejczyk** [comment]
Hi,
Error during compilation also appeared on Magento 2.2.* version, fix is available in master branch e525ba5d0dfce73b3e2140ca2135e3aba0694889
